contracts/libraries/TransferHelper.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity ^0.8.0;
 
+import 'hardhat/console.sol';
```

Comment:
get rid of references hardhat/console.sol. This is a hardhat framework reference only needed for testing. Shouldn't have this here.

Reply:
ok

Reply:
ok